src/main/java/pl/winiarczyk/lesson4/homework/JavaCollections1_2/MyList.java

```diff
@@ -0,0 +1,59 @@
+package pl.winiarczyk.lesson4.homework.JavaCollections1_2;
+
+public class MyList implements OwnList {
+
+    int[] ownList = {1, 2, 3, 4, 5};
+
+    @Override
+    public int size() {
+        return ownList.length;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        boolean isEmpty = false;
+        if (ownList.length == 0) {
+            isEmpty = true;
+            return isEmpty;
+        } else {
+            isEmpty = false;
+            return false;
+        }
+    }
+
+    @Override
+    public void add(Integer element) {
+        int n = ownList.length;
+        int newList[] = new int[n - 1];
```

Comment:
n - 1 czy n czy n + 1?? Skoro dodajemy element do listy to chyba chcemy, aby ta nowa lista była większa od poprzedniej (aby zrobić miejsce na nowy element), a nie mniejsza.